app/views/schedule_rules/index.html.haml

```diff
@@ -2,6 +2,7 @@
   = javascript_include_tag "fullcalendar.js", "schedule_rules.js"
   :javascript
     var events_path = "#{url_for(:format => 'js')}";
+    var orders_path_base = "#{facility_orders_path(current_facility)}";
```

Comment:
I feel like the schedule rules shouldn't need this as that calendar view does not include any actual reservations.